app/helpers/sessions_helper.rb

```diff
@@ -25,4 +25,13 @@ def add_food_to_cart product_id, quantity
   def current_cart
     @current_cart ||= session[:cart]
   end
+  
+  def load_products_in_cart
+    current_cart.keys
+  end
+
+  def subtotal products
+    products.reduce(0) do |sum, item| 
+                          sum + item.price * current_cart[item.id.to_s]; end
```

Comment:
```suggestion
    products.reduce(0) do |sum, item| 
      sum + item.price * current_cart[item.id.to_s]
    end
```
code tào lao chém giờ

có cần phải `;` ko, trong khi nên  xún line mới là xong